eng/versioning/version_client.txt

```diff
@@ -5,6 +5,7 @@ com.azure:azure-sdk-all;1.0.0;1.0.0
 com.azure:azure-sdk-parent;1.6.0;1.6.0
 com.azure:azure-client-sdk-parent;1.7.0;1.7.0
 com.azure:azure-ai-textanalytics;1.0.0-beta.3;1.0.0-beta.4
+com.azure:azure-ai-formrecognizer;1.0.0-beta.1;1.0.0-beta.1
```

Comment:
@samvaity @alzimmermsft @joshfree 
Shouldn't formrecognizer and textanalytics both be under an area directory named "ai" instead of in separate directories?

Reply:
@annelo-msft : Do you know if we have done this in .NET atm?

Reply:
We had a discussion about how to structure the cognitive services here: https://github.com/Azure/azure-sdk-for-net/issues/8944

The takeaway was that, given the structure of the engineering system currently, we will continue to follow the pattern of sdk/<servicename>

As Cognitive starts to unify around their four major areas, Language, Vision, etc, we may revisit this decision, but for now we will continue to follow the engineering system pattern.